test/fixture_helper.rb

```diff
@@ -0,0 +1,39 @@
+class FixtureHelper
+  def initialize(source_directories)
+    @source_directories = source_directories
+  end
+
+  def combine(destination)
+    FileUtils.rm_rf(destination) if File.directory?(destination)
```

Comment:
When I see `rm_rf` I want to be absolutely sure it is not possible to mis-call this with a wrong directory like `$HOME` or `pwd` or `/`. Can we move the `"#{Rails.root}/tmp/combined_fixtures/"` definition here so it's crystal clear when you look at it and it's actually not possible to call this with a wrong path?

Reply:
:+1